alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -156,6 +159,50 @@ def listdir(self, path):
                 f"Error when listing path {path}: error {e}"
             ) from e
 
+    def get_file_status(self, path):
+        """
+        Gets the file status of the path.
+
+        Args:
+            path (str): The full ufs path to get the file status of
+
+        Returns:
+            a dict: The dictionary has:
+                - mType (string): directory or file
+                - mName (string): name of the directory/file
+                - mPath (string): the path of the file
+                - mUfsPath (string): the ufs path of the file
+                - mLastModificationTimeMs (long): the last modification time
+                - mHumanReadableFileSize (string): the size of the human readable files
+                - mLength (integer): length of the file or 0 for directory
+
+        Example:
```

Comment:
Can you help use a struct for file status, and help update other methods as well
```
class FileStatus:
    def __init__(self, type, name, path, ufs_path, last_modification_time_ms, human_readable_file_size, length):
        self.type = type
        self.name = name
        self.path = path
        self.ufs_path = ufs_path
        self.last_modification_time_ms = last_modification_time_ms
        self.human_readable_file_size = human_readable_file_size
        self.length = length
```
```
    response.raise_for_status()
    data = json.loads(response.content)[0]
    return FileStatus(
        data['mType'],
        data['mName'],
        data['mPath'],
        data['mUfsPath'],
        data['mLastModificationTimeMs'],
        data['mHumanReadableFileSize'],
        data['mLength']
    )
```